QuantConnect.SamcoBrokerage/SamcoBrokerage.cs

```diff
@@ -965,6 +962,10 @@ private IEnumerable<Symbol> GetSubscribed()
         private void OnError(object sender, WebSocketError e)
         {
             Log.Error($"SamcoBrokerage.OnError(): Message: {e.Message} Exception: {e.Exception}");
+            if (e.Message == "The remote party closed the WebSocket connection without completing the close handshake.")
```

Comment:
Generically I think should just call Ws.Close() if outside extended market hours?